com.unity.multiplayer.mlapi/Prototyping/NetworkTransform.cs

```diff
@@ -107,6 +107,44 @@ public bool UseLocal
             set => m_UseLocal.Value = value;
         }
 
+        /// <summary>
+        /// Updates the NetworkTransform's authority model at runtime.
+        /// </summary>
+        /// <param name="newAuthority"></param>
+        public void SetAuthority(Authority newAuthority)
+        {
+            TransformAuthority = newAuthority;
+            UpdateVarPermissions();
+            // todo this should be synced with the other side. let's wait for a more final solution before adding more code here
+        }
+
+        private void UpdateVarPermissions()
+        {
+            switch (TransformAuthority)
+            {
```

Comment:
this is more than implementing tests — this adds extra functionality so it'd be `feat: something something to NetworkTransform` I believe.

Reply:
true, I'll add that to the description